add blog post for osp 1.15

chmouel opened the pull request:
Still need images and links to documentation

# Changes <!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->

<!-- Describe your changes here- ideally you can get that description straight from
your descriptive commit message(s)! -->

# Submitter Checklist

- [ ] 📝 Please ensure your commit message is clear and informative. For guidance on crafting effective commit messages, refer to the How to write a git commit message guide. We prefer the commit message to be included in the PR body itself rather than a link to an external website (ie: Jira ticket).

- [ ] ♽ Before submitting a PR, run make test lint to avoid unnecessary CI processing. For an even more efficient workflow, consider installing [pre-commit](https://pre-commit.com/) and running pre-commit install in the root of this repository.

- [ ] ✨ We use linters to maintain clean and consistent code. Please ensure you've run make lint before submitting a PR. Some linters offer a --fix mode, which can be executed with the command make fix-linters (ensure [markdownlint](https://github.com/DavidAnson/markdownlint) and [golangci-lint](https://github.com/golangci/golangci-lint) tools are installed first).

- [ ] 📖 If you're introducing a user-facing feature or changing existing behavior, please ensure it's properly documented.

- [ ] 🧪 While 100% coverage isn't a requirement, we encourage unit tests for any code changes where possible.

- [ ] 🎁 If feasible, please check if an end-to-end test can be added. See [README](https://github.com/openshift-pipelines/pipelines-as-code/blob/main/test/README.md) for more details.

- [ ] 🔎 If there's any flakiness in the CI tests, don't necessarily ignore it. It's better to address the issue before merging, or provide a valid reason to bypass it if fixing isn't possible (e.g., token rate limitations).

chmouel committed:
add blog post for osp 1.15

this is a blog post listing all features

Signed-off-by: Chmouel Boudjnah <chmouel@redhat.com>

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+---
+title: OpenShift Pipeline 1.5 PAC release
```
Here and below, shouldn't the name of the product be 1.15? And the name OpenShift Pipelines (plural).

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+OpenShift Pipeline 1.5 introduces many new features for Pipelines-as-Code, here
+is a list of the most important ones.
+
+## GitOPS Commands
```
here and below, GitOps IMO

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+
+## GitOPS Commands
+
+The GitOPS commands are those small comments you can do a on pull request to
```
"do a on" -> "do on a". IMO "you can put on a pull request" sounds better.

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+## GitOPS Commands
+
+The GitOPS commands are those small comments you can do a on pull request to
+trigger some actions on your pipelinerun via Pipelins-as-Code.
```
What actions do you mean other than triggering a pipelinerun? If this only means triggering a pipelinerun, then just make it "trigger a pipelinerun".

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+### Explicitly trigger any PipelineRuns without matching annotations
+
+It was previously only possible to trigger those PipelineRuns if they were
+matching the annotations. For examples the PipelineRuns would need to have the
```
example

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+
+That limitation has been removed and now you can trigger any PipelineRuns with
+`/test` even if it doesn't match. This is useful for example if you don't want to
+have a long running pipelinerun running and consuming resources on every
```
```suggestion
have a long running pipelinerun consuming resources on every
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+CEL equivalent) to be able to be re-triggered with `/test`.
+
+That limitation has been removed and now you can trigger any PipelineRuns with
+`/test` even if it doesn't match. This is useful for example if you don't want to
```
```suggestion
`/test` even without matching annotations. This is useful for example if you don't want to
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+
+### Explicitly trigger any PipelineRuns without matching annotations
+
+It was previously only possible to trigger those PipelineRuns if they were
```
```suggestion
Previously it was only possible to trigger the PipelineRuns with matching annotations
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+automatically exposed to the PipelineRun, like `{{ revision }}` will be
+expanded to the revision from where the event come from.
+
+The GitOPS commands now accepts arguments of the form of key=value that will
```
```suggestion
The GitOPS commands now accept arguments of the form key=value that will
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
````diff
@@ -0,0 +1,96 @@
+```
+
+It will test the pipelinerun on the main branch instead of the commit of your
+Pull Request. The only parameters that can redefine are the standard ones and
````
```suggestion
Pull Request. The only parameters you can redefine are the standard ones and
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+`pipelinesascode.tekton.dev/on-comment` This let you start a PipelineRun when
+the comment has matched the regex specified in the annotation.
+
+This let you for example define your own GitOPS command and start a PipelineRun
```
```suggestion
This allows you to define your own GitOps command and start a PipelineRun
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+the comment has matched the regex specified in the annotation.
+
+This let you for example define your own GitOPS command and start a PipelineRun
+on everything that matches the regexp
```
"everything" sounds ambiguous to me. Maybe "on every event"?
Missing "." at the end of the sentence.

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+This let you for example define your own GitOPS command and start a PipelineRun
+on everything that matches the regexp
+
+To make it even more powerful we have add a new standard parameter called `{{
```
```suggestion
To make it even more powerful we have added a new standard parameter called `{{
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+### Errors are now cleared between retest runs
+
+We had a long running bug when there is a failure with Pipelines-as-Code that
+it cannot be handled (when for example there is an issue with the cluster),
```
```suggestion
cannot be handled (for example when there is an issue with the cluster),
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+
+## Global Repository settings support
+
+There is a lot of different settings that can be set on a Repository CR, those
```
```suggestion
There are a lot of different settings that can be set on a Repository CR, those
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+
+When you run a PipelineRun you have a number of standard parameters that are
+automatically exposed to the PipelineRun, like `{{ revision }}` will be
+expanded to the revision from where the event come from.
```
```suggestion
expanded to the revision from where the event came from.
```

enarha commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+## Better error feedback on bad yaml
+
+When we could not parse the yaml file in the .tekton directory, Pipelines as
+Code used to bail out and says, there was no PipelineRuns to match.
```
```suggestion
Code used to bail out and says, there were no PipelineRuns to match.
```

savitaashture commented on docs/content/docs/blog/release-osp-1.5.md:
```diff
@@ -0,0 +1,96 @@
+trigger_comment }}` which would be available to the PipelineRun and would
+contain the whole comment that triggered the PipelineRun.
+
+### Errors are now cleared between retest runs
```
is this talking about failure checkruns ???

chmouel replied:
yes

chmouel commented:
thanks for the review @enarha the first draft wasnt that great and thanks for the suggestions, i rephrased quite a bit in there to make things clearer

chmouel committed:
Make sure to use GitOps instead of GitOPS

Using vale rule we can make sure that the term GitOps is properly
written in our documentation.

Signed-off-by: Chmouel Boudjnah <chmouel@redhat.com>

enarha commented:
/lgtm
